modules/katello_content_view.py

```diff
@@ -68,6 +67,22 @@
       - present
       - present_with_defaults
       - absent
+  auto_publish:
+    description:
+      - Auto publish composite view when a new version of a component content view is created.
+      - Also note auto publish will only happen when the component is marked "latest".
+    default: false
+    type: bool
+  composite:
+    description:
+      - A composite view contains other content views.
+    default: false
+    type: bool
+  components:
+    description:
+      - List of content views to includes name and either version or latest.
+      - Ignored if C(composite) is False.
```

Comment:
As a user of this module, I would expect that setting repository and (composite=true + components) to the same time does simply fail (= error). Additionally, I would expect to get a error, if composite=true but no components are given.
Can you please change the description + the module + add some tests to represent this behavior?

Reply:
> As a user of this module, I would expect that setting repository and (composite=true + components) to the same time does simply fail (= error).

Nice idea, I'll change that :+1: 

> Additionally, I would expect to get a error, if composite=true but no components are given.

I think that's a totally valid, I might want to create the CCV and add components later in another task? The server accepts the creation of `empty` CCV, id rather not artificially impose a restriction like this without good reason?